docs/pages/components/tag.mdx

```diff
@@ -63,46 +62,50 @@ Use size property with `xs`, `sm`, `md` or `lg`.
 <Tag size="md">
   Medium
 </Tag>
-<Tag size="lg">
-  Large
-</Tag>
 ```
 
-## One character style
+## List of clickable tags
 
-When we have only one character, we apply same size for the width and height (it implicitely sets `shape` prop to `square`)
+When we have a clickable tag, we apply an hover state
 
 ```jsx row
+<Tag size="xs" onClick="">
+  <a href="#">Some link</a>
+</Tag>
 <Tag size="sm">
-  1
+  <a href="#">Some link</a>
 </Tag>
 <Tag size="md">
-  1
-</Tag>
-<Tag size="lg">
-  1
+  <a href="#">Some link</a>
 </Tag>
 ```
 
-## Shape
+## One character style
 
-Set `shape` prop to `square` to get a square no matter the content.
+When we have only one character, we apply same size for the width and height
 
 ```jsx row
-<Tag size="lg" shape="square">
+<Tag size="xs">
+  1
+</Tag>
+<Tag size="sm">
   1
 </Tag>
-<Tag size="lg" shape="square" w={50}>
-  Long text
+<Tag size="md">
+  1
 </Tag>
 ```
 
-Set `shape` prop to `circle` to set a border-radius `50%`.
+## OnClick
+
+Simply add a function `OnClick` to catch the action with an hover effect.
```

Comment:
Hey, here it's an `onClick` function. It's not very different but I think it could be confusing

Reply:
Totally agree with you 👍